src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,222 @@
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Click
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-click.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Disabled
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-disabled.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Required
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-required.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Error
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-error.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Make the input label succinct and descriptive. Don't construct sentences or ask questions with the input label. Put conversational labels or lengthy descriptions outside the Input box and use section headings. See  <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">Form design</a> guidelines for more information.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use section headers outisde of Input box to supply conversational form design."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/input-box/input-box-content-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      <div
+        class="sky-docs-text-block"
+      >
+        <sky-docs-design-guideline>
+          <p>
+            Input boxes are most common inside Modals and follow <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">Form design</a> guidelines. When using 2-column or 3-column layouts for Input boxes, use the narrow spacing option of Fluid grid to maintain even vertical and horizontal padding.
```

Comment:
Lowercase "modals" and "form design" in the first sentence. In the second sentence, spell out two-column and three-column. Fluid grid should be lowercase.

Reply:
I've made your suggestions, but @Blackbaud-AdamFunderburk might need to speak to what was intended by the "narrow option" ...

Reply:
The changes look good but leaving this conversation open to make sure Adam sees the questions.

Reply:
I'm using "even" here to mean "balanced", so I'm fine with that change if you think it's better.

The different margin options on Fluid grid are part of the new modern theme Spacing documentation, but are not yet part of the component documentation itself. And that option is actually called `small` instead of "narrow" now that I look closer. It might be better to link straight to the Spacing guidelines instead of the Fluid grid component for now. What do you think?

Reply:
I changed "narrow" to "small" and changed the link to open the spacing guidelines instead of the fluid grid docs.

That said, I don't see a `small` option in the spacing docs or in the PR at https://github.com/blackbaud/skyux2-docs/pull/949. If "small" is the right terminology and we want to link to the spacing guidelines, this should be good to go. But I still feel like I'm missing something here.

Reply:
I talked through this some more with Alex and dug further into the modern theme docs, Based on that, I updated the paragraph to `Input boxes are most common inside modals and follow form design guidelines. For two-column or three-column layouts, use fluid grid's small gutters spacing option to apply vertical and horizontal padding evenly.`. I think that should be good to go, so I'm resolving the thread, but let me know if not.